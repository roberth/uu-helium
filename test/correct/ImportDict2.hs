module ImportDict2(A(MkA),y) where

data A a = MkA a

y = x
x = 3