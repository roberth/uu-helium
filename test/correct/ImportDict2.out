Prelude is up to date
Compiling correct/ImportDict2.hs
(5,1): Warning: Missing type signature: y :: Int
(6,1): Warning: Missing type signature: x :: Int
Compilation successful with 2 warnings
No 'main' function defined in this module
